engine/chain/internal/junglebus/txs_getter_test.go

```diff
@@ -0,0 +1,89 @@
+package junglebus_test
+
+import (
+	"context"
+	"slices"
+	"testing"
+	"time"
+
+	chainerrors "github.com/bitcoin-sv/spv-wallet/engine/chain/errors"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/internal/junglebus"
+	"github.com/bitcoin-sv/spv-wallet/engine/tester"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTransactionGetter(t *testing.T) {
```

Comment:
I would rather test the whole solution (with the combined getter, and optionally turned on or off junglebus fallback) at the level of `Chain().Broadcast`
But I'm not sure if it would be very difficult.

Reply:
After internal conversation, we decided to remove the tests and move some of the cases to `arc/broadcast_test.go` to test it from "not internally exported" `Broadcast` method.

Reply:
I've moved the junglebus mock requests to `ARC` internal package and removed the tests from `junglebus` package.
The junglebus is now tested by the way of testing ARC.

![image](https://github.com/user-attachments/assets/3223348f-f89c-45b3-9b82-c7b209ff5fe5)